app/forms/apo_form.rb

```diff
@@ -35,6 +35,7 @@ def sync
     model.mods_title           = params[:title]
     model.desc_metadata_format = params[:desc_md]
     model.metadata_source      = params[:metadata_source]
+
```

Comment:
:question:

Reply:
Oops.  I probably threw a byebug here at one time and then remove it but left the `\n`

Reply:
nbd